content/en/blog/2019/isolated-clusters/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Use Istio multi-mesh deployments for isolation and boundary protection
+subtitle: Deploy environments that require isolation into separate meshes and enable inter-mesh communication by mesh federation
+description: Deploy environments that require isolation into separate meshes and enable inter-mesh communication by mesh federation.
+publishdate: 2019-09-26
+attribution: Vadim Eisenberg (IBM)
+keywords: [traffic-management,multicluster,security,gateway,tls]
+---
+Various compliance standards require protection of sensitive data environments. Some of the important standards and the
+types of sensitive data they protect appear in the following table:
+
+|Standard|Sensitive data|
+| --- | --- |
+|[PCI DSS](https://www.pcisecuritystandards.org/pci_security)|payment card data|
+|[FedRAMP](https://www.fedramp.gov)|federal information, data and metadata|
+|[HIPAA](http://www.gpo.gov/fdsys/search/pagedetails.action?granuleId=CRPT-104hrpt736&packageId=CRPT-104hrpt736)|personal health data|
+|[GDPR](https://gdpr-info.eu)| personal data|
+
+[PCI DSS](https://www.pcisecuritystandards.org/pci_security), for example, recommends putting cardholder data
+environment on a network, separate from the rest of the system. It also requires using a [DMZ](https://en.wikipedia.org/wiki/DMZ_(computing)),
+and setting firewalls between the public Internet and the DMZ, and between the DMZ and the internal network.
+
+Isolation of sensitive data environments from other information systems can reduce the scope of the compliance checks
+and improve the security of the sensitive data. Reducing the scope reduces the risks of failing a compliance check and
+reduces the costs of compliance since there are less components to check and secure, according to compliance
+requirements.
+
+You can achieve isolation of sensitive data environments by deploying the applications that process the sensitive data
+into {{< gloss "multi-mesh" >}}separate service meshes{{< /gloss >}}, preferably on separate networks. Then you can use
+Istio to connect applications in the meshes with different compliance requirements. The process of connecting inter-mesh
+applications is called {{< gloss >}}mesh federation{{< /gloss >}}.
+
+In this blog post I describe the requirements for isolation and boundary protection, and outline the principles of
+{{< gloss >}}multi-mesh{{< /gloss >}} deployments. Finally, I report the current state of
```

Comment:
```suggestion
multi-mesh deployments. Finally, I'll describe the current state of the
```

Reply:
Addressed in https://github.com/istio/istio.io/pull/4776/commits/b47950c83e472a87ad20a0346201d2d70302d5ea and in https://github.com/istio/istio.io/pull/4776/commits/c6720beabace7a940516577ab297a4f3450043f3.